Python_Engine/Compute/Invoke.cs

```diff
@@ -34,6 +34,12 @@ public static partial class Compute
 
         public static PyObject Invoke(PyObject module, string method, Dictionary<string, object> args)
         {
+            if(!Query.IsInstalled())
+            {
+                BH.Engine.Reflection.Compute.RecordError("The Python Toolkit has not been successfully installed. Please run the InstallPythonToolkit component to install the necessary dependencies to invoke Python methods within BHoM");
+                return null;
+            }
+
```

Comment:
Message looks good! Can we add it to the other `Invoke` method as well?

Reply:
That's what I get for not scrolling!! 🤦‍♂️